dcos_installer/test_backend.py

```diff
@@ -345,6 +349,52 @@ def aws_cf_configure(s3_bucket_name, config, config_aws, tmpdir, monkeypatch):
         s3_bucket.delete()
 
 
+def test_do_configure_valid_config_no_duplicite_logging(tmpdir, monkeypatch, caplog):
+    """
+    Log messages aren't printed twice.
```

Comment:
❓ Don't do yourself a disservice! This test tests more than just that the log messages aren't printed twice. It tests that they aren't printed thrice, four times, and even seven times 😆 

I'd go with something like "Log messages are logged exactly once".

If not, fix "duplicite" in the test name to say "duplicate".